corehq/apps/accounting/utils/subscription.py

```diff
@@ -10,6 +13,19 @@
 )
 
 
+def get_account_or_404(request, domain):
```

Comment:
great idea...lol it's definitely a red flag when a private method is being imported into another app's views 🙈

Reply:
thanks! I wasn't sure if this was the best spot for this method or not. Does this `utils/subscription` seem ok?

Reply:
i think it's fine to have it here. if there are more account-specific tools then this util file can get broken up into its own section later